src/server/ssrServer.js

```diff
@@ -61,6 +61,7 @@ export function createApp({ enablePostToModuleRoutes = false } = {}) {
   app.use(forwardedHeaderParser);
 
   app.use('/_/static', express.static(path.join(__dirname, '../../build'), { maxage: '182d' }));
+  app.get('/_/ready', (req, res) => res.sendStatus(200));
```

Comment:
thought we were going to use `/_/status`

Reply:
Since it always returns a 200, `status` is a bit too specific

Reply:
lets catch up on this offline